wurst/systems/crafting/CraftingMixingPot.wurst

```diff
@@ -0,0 +1,180 @@
+
+        let netherPotion = new ItemRecipeMixing(ITEM_NETHER_POTION, 2, true, ABILITY_QM_NETHER_POTION)
+            ..addColorHerb()
+            ..addItem(ITEM_RIVER_STEM)
+            ..addItem(ITEM_RIVER_STEM)
+            ..addItem(ITEM_ATHELAS_SEED)
+            ..addItem(ITEM_ATHELAS_SEED)
+            ..generateRecipeString(ABILITY_QM_NETHER_POTION)
+
+        let drunkPotion = new ItemRecipeMixing(ITEM_DRUNKS_POTION, 2, true, ABILITY_QM_DRUNKS_POTION)
+            ..addColorHerb()
+            ..addItem(ITEM_RIVER_ROOT)
+            ..addItem(ITEM_RIVER_ROOT)
+            ..addItem(ITEM_ATHELAS_SEED)
+            ..addItem(ITEM_ATHELAS_SEED)
+            ..generateRecipeString(ABILITY_QM_DRUNKS_POTION)
+
+        let elementalShieldPotion = new ItemRecipeMixing(ITEM_ELEMENTAL_SHIELD_POTION, 2, true, ABILITY_QM_ELEMENTAL_SHIELD_POTION)
+            ..addItem(ITEM_RIVER_STEM)
+            ..addColorHerb()
+            ..addItem(ITEM_ATHELAS_SEED)
+            ..generateRecipeString(ABILITY_QM_ELEMENTAL_SHIELD_POTION)
+
+        let healingPotion = new ItemRecipeMixing(ITEM_HEALING_POTION, 1, true, ABILITY_QM_HEALING_POTION)
+            ..addItem(ITEM_ATHELAS_SEED)
+            ..addItem(ITEM_RIVER_ROOT)
+            ..generateRecipeString(ABILITY_QM_HEALING_POTION)
+
+        let manaPotion = new ItemRecipeMixing(ITEM_MANA_POTION, 1, true, ABILITY_QM_MANA_POTION)
+            ..addItem(ITEM_RIVER_STEM)
+            ..addItem(ITEM_ATHELAS_SEED)
+            ..generateRecipeString(ABILITY_QM_MANA_POTION)
+
+        let antiMagicPotion = new ItemRecipeMixing(ITEM_ANTI_MAGIC_POTION, 2, true, ABILITY_QM_ANTI_MAGIC_POTION)
+            ..addColorHerb()
+            ..addAthelasSeed()
+            ..generateRecipeString(ABILITY_QM_ANTI_MAGIC_POTION)
+
+        let cureAll = new ItemRecipeMixing(ITEM_CURE_ALL, 2, true, ABILITY_QM_CURE_ALL)
+            ..addItem(ITEM_ATHELAS_SEED)
+            ..addItem(ITEM_ATHELAS_SEED)
+            ..addItem(ITEM_ATHELAS_SEED)
+            ..generateRecipeString(ABILITY_QM_CURE_ALL)
+
+        let spiritDarkness = new ItemRecipeMixing(ITEM_SPIRIT_DARKNESS, 0, true, ABILITY_QM_SPIRIT_DARKNESS)
+            ..addItem(ITEM_RIVER_STEM)
+            ..addItem(ITEM_RIVER_ROOT)
+            ..addItem(ITEM_ATHELAS_SEED)
+            ..addItem(ITEM_ATHELAS_SEED)
+            ..generateRecipeString(ABILITY_QM_SPIRIT_DARKNESS)
+
+        let spiritWater = new ItemRecipeMixing(ITEM_SPIRIT_WATER, 0, true, ABILITY_QM_SPIRIT_WATER)
+            ..addItem(ITEM_RIVER_ROOT)
+            ..addItem(ITEM_RIVER_ROOT)
+            ..generateRecipeString(ABILITY_QM_SPIRIT_WATER)
+
+        let spiritWind = new ItemRecipeMixing(ITEM_SPIRIT_WIND, 0, true, ABILITY_QM_SPIRIT_WIND)
+            ..addItem(ITEM_RIVER_STEM)
+            ..addItem(ITEM_RIVER_STEM)
+            ..generateRecipeString(ABILITY_QM_SPIRIT_WIND)
+
+        // Complex Mixing recipes
+        // If you have 4 herbs of the same color and you execute the ItemRecipe
```

Comment:
I don't follow what you mean. Can you craft an EOB with 4 blues or not?

Reply:
No because it goes through this [function](https://github.com/island-troll-tribes/island-troll-tribes/blob/729591a4adc0868aceffa982f7438bf0b043daee/wurst/systems/crafting/ItemRecipeMixing.wurst#L102) before attempting assemble.